src/Scripts/Game/HUD/EL_BetaHud.c

```diff
@@ -121,16 +121,16 @@ class EL_BetaHud : SCR_InfoDisplay
 		m_HungerProgress.SetMaskProgress(value);
 		//m_StatChange = true;
 	}
-	
-	void OnMoneyChange(float value)
+
+	void OnMoneyChange(ResourceName prefab, int newAmount, int oldAmount)
 	{
-		if (m_MoneyIndicator)
+		if (!m_MoneyIndicator)
 		{
 			m_MoneyIndicator = TextWidget.Cast(m_PlayerStatsHUD.FindAnyWidget("m_moneyIndicator"));
 			if (!m_MoneyIndicator) return;
 		}
-		
-		m_MoneyIndicator.SetText("$ " + value); //for configurabiluity could have the $ changeable in config...
+
+		m_MoneyIndicator.SetText("$ " + EL_FormatUtils.AbbreviateNumber(newAmount)); //for configurabiluity could have the $ changeable in config...
 		//m_StatChange = true;
```

Comment:
> //m_StatChange = true;

I dont know how this works but after a change in money the UI should be visible again

Reply:
Yes, probably, but the hud will need to be redone. I just put money in there temporarily.